src/store/growthbook-context.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { GrowthBook, GrowthBookProvider } from '@growthbook/growthbook-react'
+import { getClientInformation, getDomain, getLanguage } from 'common/utility'
+import { growthbook_client_key } from 'common/constants'
```

Comment:
Just a suggestion, We can move the key constant to this file and scope it here to have less dependency on other files.